crates/frontend/src/pages/default_config.rs

```diff
@@ -480,6 +492,89 @@ pub fn get_bread_crums(key_prefix: Option<String>) -> Vec<BreadCrums> {
     bread_crums
 }
 
+#[component]
+fn default_config_filter_widget(
+    filters_rws: RwSignal<DefaultConfigFilters>,
+    key_prefix: RwSignal<Option<String>>,
+) -> impl IntoView {
+    let filters = filters_rws.get_untracked();
+    let filters_buffer_rws = create_rw_signal(filters.clone());
+    view! {
+        <DrawerBtn
+            drawer_id="default_config_filter_drawer".into()
+            style="cursor-pointer btn btn-purple-outline m-1 w-[8rem]".to_string()
+        >
+            Filters
+            <i class="ri-filter-3-line"></i>
+        </DrawerBtn>
+        <Drawer
+            id="default_config_filter_drawer".to_string()
+            header="default_config Filters"
+            drawer_width="w-[50vw]"
+            handle_close=move || {
+                close_drawer("default_config_filter_drawer");
+            }
+        >
+            <div class="card-body">
+                <div class="flex flex-col gap-9 justify-between">
+                    <div class="form-control">
```

Comment:
can we please merge these two divs
```suggestion
                <div class="form-control flex flex-col gap-9 justify-between">
```